docs/.vuepress/components/FaqEntry.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <cdr-accordion
+    :id="generatedId"
+    level="3"
+    :opened="opened"
+    @accordion-toggle="opened = !opened"
+  >
+    <template #label>
+      {{ title }}
+    </template>
+    <slot/>
+  </cdr-accordion>
+</template>
+
+<script>
+export default {
+  name: 'FaqEntry',
+  props: {
+    title: { type: String, default: undefined },
+  },
+  data() {
+    return {
+      opened: false,
+    }
+  },
+  computed: {
+    generatedId() {
+      return this.title.replace(/\s+/g, '-').toLowerCase();
+    }
+  }
+}
```

Comment:
Created this wrapper for CdrAccordion to make it easier and less messy to add to the FAQ page.